admin/sql/create_tables.sql

```diff
@@ -20,6 +20,7 @@ CREATE TABLE "user" (
   contact_email    CHARACTER VARYING NOT NULL,
   data_usage_desc  TEXT,
   org_name         CHARACTER VARYING,
+  logo_filename    CHARACTER VARYING,
```

Comment:
Ah, I see. You've depricated the old logo URL, but still kept it. I think this complicated the code unnecessarily and we should get rid of the old org_logo_url and write a script that moves the legacy supporters into the new format. Then drop the old column.